tests/integration/components/ember-popper/render-in-place-test.js

```diff
@@ -24,16 +24,20 @@ test('false: renders in the body', function(assert) {
 });
 
 test('false with an explicit popperContainer: renders in the popperContainer', function(assert) {
+  this.set('show', false);
   this.render(hbs`
     <div class='poppers-plz'>
     </div>
-
+    {{#if show}}
     <div>
       {{#ember-popper-targeting-parent class='hello' popperContainer='.poppers-plz' renderInPlace=false}}
         template block text
       {{/ember-popper-targeting-parent}}
     </div>
+    {{/if}}
   `);
+  // ensure the container is in DOM before rendering the popper element
+  this.set('show', true);
```

Comment:
Note that this test was failing for Ember 2.8 and below, i.e. versions that used the ember-wormhole based polyfill instead if the native `-in-element`. It threw [this assertion](https://github.com/kybishop/ember-popper/blob/master/addon/components/ember-popper-base.js#L296-L298).

Don't know if this can be seen as a regression or breaking change? But for me this seems reasonable though to assume the container div must be already present in DOM before rendering the popper element.